java/adapter/orc/pom.xml

```diff
@@ -103,7 +103,7 @@
     <build>
         <resources>
             <resource>
-                <directory>${arrow.cpp.build.dir}/src/jni/orc</directory>
+                <directory>${arrow.cpp.build.dir}</directory>
```

Comment:
rename to cpp.binary.dir instead ? so that we do not have the confusion on the directory to pass

Reply:
too close to the release. i'll do that in a different PR after 0.14